### ISSUE TITLE
Assignments should be placed on same line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#532]
We should report a violation in case an assignment is placed on different lines.

For example, following code should report a violation:
```c#
var x = 
        42;
```
while following code should NOT report a violation:
```c#
var x = 42;
```

**Comment by RalfKoban** [comment]
Simple assignments such as
```c#
i =
    42;
```
should also be reported.
